lib/nexmo_developer/spec/services/translator/file_translator_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Translator::FileTranslator do
+  it 'accepts a documentation path to initialize an instance' do
+    doc_path = 'voice/voice-api/guides/numbers.md'
```

Comment:
you can re-use this definition in every spec by using [let](https://relishapp.com/rspec/rspec-core/v/3-9/docs/helper-methods/let-and-let)

let(:doc_path) { ''voice/voice-api/guides/numbers.md'' }

Reply:
and use `subject` with the translator, 
`subject { described_class.new(doc_path) } `

then in each spec call `subject.mthod`